app/dashboard/views.py

```diff
@@ -1036,10 +1036,6 @@ def quickstart(request):
     return TemplateResponse(request, 'quickstart.html', {})
 
 
-class ProfileHiddenException(Exception):
```

Comment:
Remove secondary definition of this exception, so there is **one** version of this exception to be handled.